content/kotsadm/installing/proxies.md

```diff
@@ -0,0 +1,19 @@
+---
+date: 2020-07-30
+linktitle: "Proxies"
+title: Installing Behind a Proxy
+weight: 10021
+draft: false
+---
+
+### Proxies
```

Comment:
This doc is short, what do you think about just adding it as a new section in the existing "online install" doc instead of creating a new doc?